src/style.css

```diff
@@ -0,0 +1,25 @@
+.body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
```

Comment:
pay attention to the last star in each row 😉

<img width="288" alt="image" src="https://user-images.githubusercontent.com/55802630/172814732-200e49ef-5ffd-476d-8ac8-c9761baff2a5.png">

Reply:
ok, thanks